src/modules/navigation/services/navigation-service/navigation.types.ts

```diff
@@ -1,6 +1,19 @@
-export interface Navigation {
-	description: string;
+export enum NavigationPlacement {
+	HeaderLeft = 'header_left',
+	FooterCenter = 'footer_center',
+}
+
+export interface NavigationInfo {
```

Comment:
Personally not a fan of the `Info`-suffix, doesn't make the name more descriptive imo

Reply:
agreed, this is mostly done because there is already a Navigation component, and IDE's would then try to import the wrong one.
Also in the proxy there was a naming conflicts between what we load from the database "Navigation" and what we send to the client "NavigationInfo". So i kept the name consistent between proxy and client.